tests/should_succeed/traits/18_trait_laws.rs

```diff
@@ -1,7 +1,8 @@
 extern crate creusot_contracts;
 use creusot_contracts::*;
 
-pub trait Symmetric {
+pub trait Symmetric: Sized {
+    // TODO
```

Comment:
What can I do here? I don't understand the error I got before I added that bound.

Reply:
The solution is probably to make every pearlite function return a reference, which we could dereference after each function call. I don't know how disruptive this is.

Otherwise, I think it's fair to add a `: Sized` bound here....

But, out of curiosity, why do we have these `*&` everywhere?